js/common.js

```diff
@@ -268,3 +169,26 @@ function FixToc() {
         }
     }
 }
+
+//Animation header on scroll
+function FixHead() {
+    var header = $('#header');
+    if (header.length > 0) {
+        var headerFixPosition = $(".nav-hero-container").innerHeight();
+        if (window.pageYOffset > headerFixPosition) {
+            header.addClass("not-top"); //When navigation below offset
+            header.addClass("pinned"); //When navigation below hero section
+            header.removeClass("unpinned");
+        }
+        else {
+            header.removeClass("pinned");
+            header.addClass("unpinned");
+        }
+
+        //Return classes to the initial state when the navigation at the top of the page
+        if (window.pageYOffset < 64) {
```

Comment:
Could you take this value — 64  by JS from container  `.innerHeight();` ? To make it more flexible for styles changes. 

For this `tocNav.tocify({selectors:"h2, h3, h4", showAndHide: false, scrollTo: 64, extendPage: false});` place as well.